java/src/main/java/vowpalWabbit/learner/VWLearners.java

```diff
@@ -78,28 +102,24 @@ private static long initializeVWJni(final String command) {
         }
         return nativePointer;
     }
-
+    
     private static void loadNativeLibrary() {
         // By making use of a static lock here we make sure this code is only executed once globally.
         if (!loadedNativeLibrary) {
             STATIC_LOCK.lock();
             try {
                 if (!loadedNativeLibrary) {
-                    NativeUtils.loadOSDependentLibrary("/vw_jni", ".lib");
+                    System.loadLibrary("vw_jni");
                     loadedNativeLibrary = true;
                 }
             }
-            catch (IOException e) {
-                // Here I've chosen to rethrow the exception as an unchecked exception because if the native
-                // library cannot be loaded then the exception is not recoverable from.
-                throw new RuntimeException(e);
-            }
             finally {
                 STATIC_LOCK.unlock();
             }
         }
     }
     private static native long initialize(String command);
+    private static native long seedVWModel(long nativePointer);
```

Comment:
Lets say you have a vw model of size 1GB and you create 2 instances of the same model using initialize, memory footprint in java is 2 * 1GB. Now if you use seed_vw_model, internal data structures are shared and hence memory footprint would still be 1GB.

We still need initialize to create the first instance. seed_vw_model requires a seed model to clone from.